content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,124 @@
+the amounts to be specified for each product, and what pack(s) the product(s) will go into. Add a
+line for each additional package to be used, and specify the amount of the product to go in the
+specific package. Once all the products for each line has been entered, click :guilabel:`Confirm`.
+Repeat this for all products on the delivery order.
+
+.. image:: usage/packages-detailed-14-15.png
+   :align: center
+   :alt: Detailed operations where the amount of product going in a pack can be specified.
+
+The done quantity on the delivery order will be updated to show the products selected in the various
+packages. When the done amount matches the demand amount, click :guilabel:`Put In Pack`, then
+:guilabel:`Validate` the delivery order. Clicking on the :guilabel:`Packages` smart button will show
+the packages used in the delivery order.
+
+.. image:: usage/packages-14-15-out.png
+   :align: center
+   :alt: Put in pack once the done amount matches the demand.
+
+Packagings
+==========
+
+*Packaging* is product specific, and refers to a disposable container that holds several units of a
+specific product. Unlike packages, packagings cannot be reusable, and each specific packaging must
+be defined on the individual product form. For example, different packages for cans of soda can be
+configured as a 6-pack, a 12-pack, or a case of 36. Each flavor of soda would need a 6, 12, and 36
+can packaging configured on the individual product since packagings are product specific, not
+generic.
+
+Create different packaging for a product
+----------------------------------------
+
+To use packagings, ensure :guilabel:`Product Packagings` is enabled under :menuselection:`Inventory
+--> Configuration --> Settings --> Products`. In Odoo, product packagings are used on sales/purchase
+orders and inventory transfers.
+
+To create packagings, click :guilabel:`Edit` on the product page. In the inventory tab, click
+:guilabel:`Add a line`. Enter the information for each packaging, specifying the name, type of
+packaging, and the amount in each pack, then click :guilabel:`Save`.
+
+.. image:: usage/grape-soda-14.png
+   :align: center
+   :alt: Packaging specified on the product page form inventory tab.
+
+View all packagings
+-------------------
+
+To view all packagings that have been created, go to :menuselection:`Inventory --> Configuration -->
+Product Packagings`, and a list will appear with all packagings created for all products. In this
+example, there are two different kinds of sodas with three types of packagings configured for each.
+New packagings can be created from this report by clicking :guilabel:`Create`.
+
+.. image:: usage/packagings-14.png
+   :align: center
+   :alt: List of different packagings for products.
+
+When creating a sales order, specify the packagings that should be used for the product. In this
+example, 18 cans of soda will be picked and/or packed using three 6-pack packagings.
```

Comment:
When creating a sales order in the :guilabel:`Sales` application, specify the packagings that should be used for a product(s).

In this example, 18 cans of soda will picked and/or packed using three 6-pack packagings.